tests/lib/rules/jsx-no-literals.js

```diff
@@ -226,7 +240,7 @@ ruleTester.run('jsx-no-literals', rule, {
           }
         }
       `,
-      options: [{noStrings: true, allowedStrings: ['foo: ', '*']}]
+      options: [{noStrings: true, allowedStrings: ['foo:', '*']}]
```

Comment:
this is either a backwards incompatibility or a bugfix, depending on perspective. my vote is the latter based on the conflict with error messages.

Reply:
would passing in `foo: ` (with the space) no longer work?

Could/should we trim all the allowedStrings before checking?

Reply:
ah, of course! running `allowedStrings` through trim would be a simple way to make this backwards compatible.

Reply:
done